text/0000-inline-asm.md

```diff
@@ -823,8 +823,12 @@ unsafe fn foo(mut a: i32, b: i32) -> (i32, i32)
     - You cannot exit an `asm!` block that has not been entered. Neither can you exit an `asm!` block that has already been exited.
     - You are responsible for switching any target-specific state (e.g. thread-local storage, stack bounds).
     - The set of memory locations that you may access is the intersection of those allowed by the `asm!` blocks you entered and exited.
+- You cannot assume that an `asm!` block will appear exactly once in the output binary. The compiler is allowed to instantiate multiple copies of the `asm!` block, for example when the function containing it is inlined in multiple places.
```

Comment:
Is it safe to make such assumptions if you use `inline(never)`?

Reply:
No. Use `global_asm!` if you need to define symbols.